### ISSUE TITLE
LWG3639: Handling of fill character width is underspecified in std::format

### ISSUE THREAD

**Issue by tahonermann** [cplusplus/papers#1150]
SG16 needs to clarify the intended design for https://cplusplus.github.io/LWG/issue3639.

```
 20.20.2.2 [format.string.std] doesn't specify if implementations should consider the estimated width of the fill character when substituting it into the formatted results.

For example:

    auto s = std::format("{:🤡>10}", 42);

"🤡" (U+1F921) is a single code point but its estimated display width is two.

There are at least three possible resolutions:

    1. s == "🤡🤡🤡🤡42": use the estimated display width, correctly displayed on compatible terminals.

    2. s == "🤡🤡🤡🤡🤡🤡🤡🤡42": assume the display width of 1, incorrectly displayed.

    3. Require the fill character to have the estimated width of 1.
```

**Comment by tahonermann** [comment]
SG16 discussed (a draft of) [P2572R0](https://wg21.link/p2572r0) at its [2022-06-08 telecon](https://github.com/sg16-unicode/sg16-meetings#june-8th-2022). The following poll was taken:

  - **Poll 1: Revise D2572R0 "std::format() fill character allowances" as discussed,
    and forward the paper so revised to LEWG as the recommended resolution of LWG3576
    and LWG3639.**
    - Attendance: 8 (2 abstention)

      | SF  | F   | N   | A   | SA  |
      | --: | --: | --: | --: | --: |
      |   3 |   3 |   0 |   0 |   0 |

    - Consensus in favor.

The changes in the published [P2572R0](https://wg21.link/p2572r0) revision reflect the SG16 requested changes.

Removing the SG16 label; this is ready for LWG pending approval of issue #1260 and (a revision of) [P2572](https://wg21.link/p2572) by LEWG.

**Comment by frederick-vs-ja** [comment]
This should have been fixed by cplusplus/draft#6144.
